### DISCUSSION TITLE
[Question][Grade] Can I know the final letter grade criteria?

### DISCUSSION THREAD

**Discussion by KAIST-JongchanPark** [prosyslab-classroom/cs348-information-security#639] [Q&A]
I think I got too low final letter grade than I expected, considering my assignment scores and average assignment scores.

Can I know the final letter grade criteria?

**Comment by KihongHeo** [comment]
If you have any question, directly email me and Minsuk.

**Comment by KAIST-JongchanPark** [comment]
Thanks!
